plugin.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: wp-gutenberg-epfl
  * Description: EPFL Gutenberg Blocks
  * Author: WordPress EPFL Team
- * Version: 1.9.0
+ * Version: 1.9.1
```

Comment:
C'est pour moi un changement mineur, pas un bugfix
```suggestion
 * Version: 1.10.0
```